create-and-save-map/src/main/java/com/esri/arcgismaps/sample/createandsavemap/screens/MainScreen.kt

```diff
@@ -0,0 +1,462 @@
+                            ) {
+                                Text(text = "Name: ", Modifier.padding(8.dp))
+                                TextField(
+                                    value = mapViewModel.mapName,
+                                    onValueChange = { newName -> mapViewModel.updateName(newName) },
+                                    Modifier.width(270.dp),
+                                    singleLine = true
+                                )
+                            }
+                            Spacer(Modifier.size(8.dp))
+
+                            Row(
+                                Modifier.fillMaxWidth(),
+                                verticalAlignment = Alignment.CenterVertically,
+                                horizontalArrangement = Arrangement.SpaceBetween
+                            ) {
+                                Text(text = "Tags: ", Modifier.padding(8.dp))
+                                TextField(
+                                    value = mapViewModel.mapTags,
+                                    onValueChange = { newTags -> mapViewModel.updateTags(newTags) },
+                                    Modifier.width(270.dp),
+                                    singleLine = true
+                                )
+                            }
+                            Spacer(Modifier.size(8.dp))
+
+                            FolderDropdown(
+                                mapViewModel.portalFolder,
+                                mapViewModel.portalFolders,
+                                mapViewModel::updateFolder
+                            )
+                            Spacer(Modifier.size(8.dp))
+
+                            Row(
+                                Modifier.fillMaxWidth(),
+                                verticalAlignment = Alignment.CenterVertically,
+                                horizontalArrangement = Arrangement.SpaceBetween
+                            ) {
+                                Text(text = "Description: ", Modifier.padding(8.dp))
+                                TextField(
+                                    value = mapViewModel.mapDescription,
+                                    onValueChange = { newText ->
+                                        mapViewModel.updateDescription(
+                                            newText
+                                        )
+                                    },
+                                    Modifier.width(270.dp),
+                                    placeholder = { Text(text = "Enter a description here.") },
+                                    keyboardOptions = KeyboardOptions(imeAction = ImeAction.Done)
+                                )
+                            }
+                            Spacer(Modifier.size(8.dp))
+
+                            Button(
+                                modifier = Modifier.align(Alignment.End),
```

Comment:
Let's center align the "Save to account" button
```suggestion
                                modifier = Modifier
                                    .align(Alignment.CenterHorizontally)
                                    .padding(12.dp),
```